pandas-docs/source/index.rst.template

```diff
@@ -51,6 +51,12 @@ See the :ref:`overview` for more detail about what's in the library.
     whatsnew/index
 {% endif -%}
 
+.. toctree::
```

Comment:
This is an extra toctree item with a caption, to test both:

* Are multiple toctree items parsed
* Are toctree items with a caption parsed